cheat/cheatsheets/pip

```diff
@@ -17,5 +17,8 @@ pip list --outdated
 # Upgrade all outdated packages, thanks to http://stackoverflow.com/a/3452888
 pip freeze --local | grep -v '^\-e' | cut -d = -f 1 | xargs -n1 pip install -U
 
+# UPgrade all outdated packages (shorter command)
```

Comment:
No, that's a typo (a.k.a. "Shift happens"). I'm sorry, should've paid more attention.